tests/ffi/tests.cc

```diff
@@ -206,6 +206,16 @@ void c_take_ref_rust_vec(const rust::Vec<uint8_t> &v) {
   }
 }
 
+void c_take_ref_rust_vec_copy(const rust::Vec<uint8_t> &v) {
+  // Test vector copy which was not compiling under gcc 7
```

Comment:
Please omit this comment, or change the comment to explain that this exercises the requirements of InputIterator.

Reply:
Put in a better comment ae95e07